docs: add all mimetypes instructions

indraneel12 opened the pull request:
added instruction to preferably customize this extension to work on all file types

indraneel12 committed:
updated README.md

added instruction to preferably customize this extension to work on all file types

Merrit commented:
Could you clarify the use case for such a change?

If I want to be able to open a specific file type in VS Code, I just associate the mimetype. Then it is either the default app to open, or available in the additional `Open with...` options.

indraneel12 commented:
Yes @Merrit, I can confirm a use-case apropos of my experience with increased productivity.

As I wanted to associate _all_ mimetypes with a particular application (VS Code), which was not merely to associate few specific filetype(s) with VS Code, I would have had to manually/programmatically associate each and every mimetype to VS Code if I chose to do it by custom mimetype association. Thanks to your Extension that saved me time by offering a direct Context Menu option, which I tweaked to fit my use-case (and chose to share it as enhancement).

I agree this is not necessarily a popular use-case, which is why I updated just the README file as `## Customization`. Hopefully, it shall help even those users who may want to additionally associate VS Code with all mimetypes for their own reasons.

Merrit commented on README.md:
```diff
@@ -24,6 +24,10 @@ Set as executable:
 
+## Customization
+
+If you prefer the extension to work on all file types, edit the file `openVSCode.desktop` by replacing the line `MimeType=inode/directory` with `MimeType=all/all`.
```
Small suggested change for clarity.

```suggestion
By default the extension works on directories. If you prefer the extension to work on all file types, edit the file `openVSCode.desktop` by replacing the line `MimeType=inode/directory` with `MimeType=all/all`.
```

indraneel12 committed:
updated README.md

Incorporated additional clarity suggested by the Repo Owner w.r.t. Customization.

Co-authored-by: Kristen McWilliam <9575627+Merrit@users.noreply.github.com>

indraneel12 committed:
updated README.md

minor attention to detail

Merrit commented:
LGTM. Thanks!
